pkg/entity/data_connector/azure_blob_storage.go

```diff
@@ -44,7 +45,7 @@ func createAzureBlobStorageCmd() *cobra.Command {
 					},
 				},
 			}
-			create(dataConnector)
+			create(dataConnector, nil)
```

Comment:
This will result in nil pointer dereference. You should pass the `cmd` arg here.

Reply:
Don't know how this got in though